cmd/mirror-url.go

```diff
@@ -113,10 +113,11 @@ func deltaSourceTarget(sourceURL, targetURL string, isFake, isOverwrite, isRemov
 
 	// List both source and target, compare and return values through channel.
 	for diffMsg := range objectDifference(sourceClnt, targetClnt, sourceURL, targetURL) {
+
 		if diffMsg.Error != nil {
-			errorIf(diffMsg.Error, "Unable to mirror objects.")
-			// Ignore error and proceed to next object.
-			continue
+			// Send all errors through the channel
+			URLsCh <- URLs{Error: diffMsg.Error}
+			return
```

Comment:
@ebozduman if we are returning here, it means we are not doing any objectDifference() anymore. So if one of these errors occur `BrokenSymlink, TooManyLevelsSymlink, PathNotFound, PathInsufficientPermission, ObjectOnGlacier`.. the code is not ignoring errors, it just stops without showing any error to the user.

What do you think ?

Reply:
agree with @vadmeste - shouldn't this be a continue instead of return statement?

Reply:
Will replace `return` with `continue`.

Reply:
Rebased.